controller/volunteer_controller.py

```diff
@@ -9,22 +10,34 @@ def create_volunteer(username: str,
                      lastname: str,
                      phone: str,
                      camp: Camp) -> Volunteer:
-    """
-    Yunsy, Yingbo
-    """
-    pass
+    try:
+        volunteer_called = Volunteer(username=username, password=password, firstname=firstname, lastname=lastname,
+                                     phone=phone)
+        camp.volunteers.add(volunteer_called)
+        return volunteer_called
+    except Volunteer.InvalidUsernameException:
+        raise ControllerError(f"Invalid username: {username}. Username should be at least 4 characters.")
+    except Volunteer.InvalidPasswordException:
+        raise ControllerError(f"Invalid password: {password}. Password should be at least 4 characters.")
+    except Volunteer.InvalidFirstnameException:
+        raise ControllerError(f"Invalid name: {firstname}. First name should be more than 1 character.")
+    except Volunteer.InvalidLastnameException:
+        raise ControllerError(f"Invalid name: {lastname}. Last name should be more than 1 character.")
+    except Volunteer.InvalidPhoneException:
+        raise ControllerError(
+            f"Invalid phone number: {phone}. Phone number should start with a plus sign and international code")
```

Comment:
Since the messages are in the original exceptions already, I think it can be simplified to:
```python
    except (Volunteer.InvalidUsernameException, 
            Volunteer.InvalidPasswordException, 
            Volunteer.InvalidFirstnameException,
            Volunteer.InvalidLastnameException, 
            Volunteer.InvalidPhoneException) as e:
        raise ControllerError(str(e))
```
So that if in the future if you change the error message in the Volunteer exceptions, you don't need to change them here too.